doc/classes/DisplayServer.xml

```diff
@@ -2246,6 +2246,25 @@
 				Makes the window specified by [param window_id] request attention, which is materialized by the window title and taskbar entry blinking until the window is focused. This usually has no visible effect if the window is currently focused. The exact behavior varies depending on the operating system.
 			</description>
 		</method>
+		<method name="window_set_taskbar_progress_value">
+			<return type="void" />
+			<param index="0" name="value" type="int" />
+			<param index="1" name="total" type="int" />
+			<param index="2" name="window_id" type="int" default="0" />
+			<description>
+				Create a progress bar on taskbar icon of the window specified by [param window_id] if not exists, set the progress of the taskbar icon.
+				[b]Note:[/b] This method is implemented only on Windows.
+			</description>
```

Comment:
I think the API would be easier to understand and use if it only had a single float `value` from `0..1`. Values outside the range would be clamped.
Internally, you multiply the value by a constant like `100000` to convert it to integer, and also use that constant for the `total`.

Same change for `Window.set_taskbar_progress_value`. Documentation must also be updated to mention the `0..1` range.